packages/flex-plugin-scripts/src/scripts/__tests__/check-start.test.ts

```diff
@@ -269,4 +276,56 @@ describe('check-start', () => {
       expect(exit).not.toHaveBeenCalled();
     });
   });
+
+  describe('_checkPluginCount', () => {
+    const _readIndexPage = jest
+      .spyOn(checkStartScript, '_readIndexPage');
+
+    beforeEach(() => {
+      _readIndexPage.mockReset();
+    })
+
+    afterAll(() => {
+      _readIndexPage.mockRestore();
+    })
+
+    it('should check with no error', () => {
```

Comment:
remove this test? the one below this seems to cover it

Reply:
Ya will remove